<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hotel</class>
 <widget class="QMainWindow" name="Hotel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>248</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>741</width>
    <height>248</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>742</width>
    <height>248</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Hotel Order Menu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f3e8d5;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="EatButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
  border: 2px transparent;
  border-radius: 10px;
  padding: 0 8px;
  background: #bc663f;
 color: White
}
QPushButton:hover {
background-color: #ffc836; 
color: White;              
}</string>
    </property>
    <property name="text">
     <string>Eat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PlayButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
  border: 2px transparent;
  border-radius: 10px;
  padding: 0 8px;
  background: #bc663f;
 color: White
}
QPushButton:hover {
background-color: #008000; 
color: White;              
}</string>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SleepButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
  border: 2px transparent;
  border-radius: 10px;
  padding: 0 8px;
  background: #bc663f;
 color: White
}
QPushButton:hover {
background-color: #101a4d; 
color: White;              
}</string>
    </property>
    <property name="text">
     <string>Sleep</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OutButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
  border: 2px transparent;
  border-radius: 10px;
  padding: 0 8px;
  background: #bc663f;
 color: White
}
QPushButton:hover {
background-color: Red; 
color: White;              
}</string>
    </property>
    <property name="text">
     <string>Leave Room</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ViewAnswers">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>160</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>View Answers</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Payment">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>100</width>
      <height>52</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>100</horstretch>
      <verstretch>52</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>52</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/images/fb-share.png);
border-color: Transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Payment_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>111</width>
      <height>52</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>100</horstretch>
      <verstretch>52</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>52</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/images/Google_Pay_Logo.png);
background-color: rgb(255, 255, 255);
border-color: Transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="LogOut">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>170</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
  border: 2px transparent;
  border-radius: 10px;
  padding: 0 8px;
  background: transparent;
  color: Red;
}
QPushButton:hover {
background-color: Red; 
color: White;              
}</string>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
